clustered/dist/build.gradle

```diff
@@ -51,21 +51,58 @@ apply plugin: EhDistribute
 
 dependencies {
   shadow "org.slf4j:slf4j-api:$parent.slf4jVersion"
-  shadowProvided 'javax.transaction:jta:1.1', 'org.codehaus.btm:btm:2.1.4'
+  shadowProvided "org.ehcache:ehcache:$parent.baseVersion"
```

Comment:
This fails as this is not really a public dependency, rather a build production. There may need to be a slightly different pom mangling for that one dep.